bundles/automation/org.eclipse.smarthome.automation.providers/src/main/java/org/eclipse/smarthome/automation/core/provider/ModuleTypeResourceBundleProvider.java

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH
+ * http://www.prosyst.com
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    ProSyst Software GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.smarthome.automation.core.provider;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+
+import org.eclipse.smarthome.automation.handler.parser.Parser;
+import org.eclipse.smarthome.automation.handler.provider.ModuleTypeProvider;
+import org.eclipse.smarthome.automation.type.ModuleType;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+
+/**
+ * @author Ana Dimova - Initial Contribution
+ *
+ */
+public class ModuleTypeResourceBundleProvider extends GeneralResourceBundleProvider implements ModuleTypeProvider {
+
+    /**
+     * @param bc
+     */
+    public ModuleTypeResourceBundleProvider(BundleContext bc) {
+        super(bc);
+        path = PATH + "/module.types/";
```

Comment:
I don't really like the "." in the path. I would prefer "module-types" or "moduletypes".